CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ As a minor extension, we have adopted a slightly different versioning convention
 
 - `mithril-aggregator` node produces artifact for different signed entity types in parallel.
 
+- Implement the signable and artifact builders for the signed entity type `CardanoStakeDistribution`
```

Comment:
Could you put that bullet point under a `- **UNSTABLE** Cardano stake distribution certification:` block?

Reply:
Done ✅